qiskit/dagcircuit/dagcircuit.py

```diff
@@ -1828,7 +1857,7 @@ def properties(self):
         }
         return summary
 
-    def draw(self, scale=0.7, filename=None, style="color"):
+    def draw(self, scale=0.7, filename=None, style="color", **kwargs):
```

Comment:
Can we shift this to a different PR? It seems unrelated.

Reply:
removed in ff987f1c6